pgxn/neon/neon.control

```diff
@@ -1,6 +1,6 @@
 # neon extension
 comment = 'cloud storage for PostgreSQL'
-default_version = '1.5'
+default_version = '1.6'
```

Comment:
BTW, should you also modify some tests that did a manual update before? (iirc, there should be some related to prewarm and co)

Reply:
I'll update the tests in https://github.com/neondatabase/neon/pull/12183 for smaller diff here

Reply:
I see that many tests failed, so not sure you can do that